packages/components/pie-toast/test/component/pie-toast.spec.ts

```diff
@@ -192,5 +196,45 @@ test.describe('PieToast - Component tests', () => {
                 expect(actionLeading).toBeVisible();
             });
         });
+
+        test.describe('variant', () => {
+            variants.forEach((variant) => {
```

Comment:
I think it may be better to split this test up into two sections and then filter out the `neutral` variant in each so we dont need the `if` statement.

e.g

```javascript
test.describe('Variant')
     test.describe('Variants with icons') => {...
        const iconVariants = variants.filter(variant => variant !== 'neutral');
        iconVariants.forEach((variant) => {....
        test(`should show the ${variant} icon`)

    test.describe('Variants without icons') => {...
        const noIconVariants = variants.filter(variant => variant === 'neutral');
        noIconVariants.forEach((variant) => {....
        test(`should not show a icon when ${variant} is set`)
```

Reply:
I am not sure, using the suggestion above I need to loop in the variants array twice.

Reply:
I think that's ok tbh, we usually stay away from `if` statements in tests because it adds a layer on top of the tests and makes it harder to read.